scripts/jointLklDM.C

```diff
@@ -873,7 +873,7 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
       //////////////////////////////////////////////////////////////
       if (exportData)
         for (Int_t isv=0; isv<=svNPoints; isv++)
-          vlkl2D[isv][imass] = grLklParabola[imass]->Eval(svScan[isv]) - grLklParabola[imass]->Eval(0);
+          vlkl2D[isv][imass] = grLklParabola[imass]->Eval(svScan[isv]);
```

Comment:
Ok, you know better than me, adopted, but in the .cc file, since the macro will be deprecated in the next release

Reply:
It's just the recent (at least few months) change of convention to not bound the lkl curve to zero at <sigmav>=0 but at the minimum of the parabola. Same as above, shouldn't we have it also in the macro so the last version of it is in agreement with the adopted convention?

Reply:
Ok